### PR TITLE
LPS-187093 Fix excessive API call in relationship modal

### PR DESCRIPTION
**Pull request by danvb112**
Related Issue: https://issues.liferay.com/browse/LPS-187093

### CODE REVIEW

**Comment by naraandrade** [comment]
Just started reviewing :)

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectRelationship/ObjectRelationshipFormBase.tsx [hunk 1]
```diff
@@ -29,6 +29,21 @@ import React, {useEffect, useMemo, useState} from 'react';
 
 import {defaultLanguageId} from '../../utils/constants';
 
+interface UseObjectRelationshipFormProps {
+	initialValues: Partial<ObjectRelationship>;
+	onSubmit: (relationship: ObjectRelationship) => void;
+	parameterRequired: boolean;
+}
+
+interface UseObjectRelationshipFormPros {
```
Comment (naraandrade) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectRelationship/ObjectRelationshipFormBase.tsx:38]: @danvb112 Please, rename this interface to "ObjectRelationshipFormBaseProps" to keep the pattern. Remember  to move its declaration above to follow the alphabetical order. Thanks :)

**Review by naraandrade — CHANGES_REQUESTED** [review]

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectRelationship/ObjectRelationshipFormBase.tsx (hunk 1 above)
Comment (danvb112) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectRelationship/ObjectRelationshipFormBase.tsx]: Hey @naraandrade, thanks for noticing it! clearly i just copied the name to the other one and completely forgot to change :rofl:

**Comment by danvb112** [comment]
ci:test:sf

**Comment by danvb112** [comment]
ci:test:relevant

**Review by naraandrade — APPROVED** [review]

**Comment by naraandrade** [comment]
ci:forward

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by danvb112** [comment]
Sent to: https://github.com/brianchandotcom/liferay-portal/pull/136694

**Pull request closed without merging**